src/context/auth-context.jsx

```diff
@@ -45,6 +46,7 @@ function AuthProvider(props) {
     signup,
     lgoinRecruiter,
     recruiter,
+    navigate,
```

Comment:
I think there is no need to create a navigation constant and then pass it there in the object value because you can create and use the action in any component.

Maybe there is some motivation to use it here, can you show me why? 🤔